apps/server/src/shared/domain/entity/import-user.entity.ts

```diff
@@ -35,6 +35,9 @@ export enum RoleName {
 }
 
 @Entity({ tableName: 'importusers' })
+@Unique({ properties: ['school', 'ldapId'] })
```

Comment:
Interesting. This works because `@Unique` is basically `@Index` without index options.
https://github.com/mikro-orm/mikro-orm/blob/7ce12d6f54845d169c769084d90ec82a1ab15c35/packages/mongodb/src/MongoSchemaGenerator.ts#L109 
For me that was not really clear from the documentation. But it's logical.